src/structures/BaseMessageComponent.js

```diff
@@ -57,7 +57,7 @@ class BaseMessageComponent {
    * @returns {?MessageComponent}
    * @private
    */
-  static create(data, client, skipValidation = false) {
+  static create(data, client) {
```

Comment:
With the removal of `skipValidation`, I believe the `@param` for this should also go on line 56, right?

Reply:
Yeah thanks, I always miss docs stuff